waspc/src/Wasp/Generator/Node/Version.hs

```diff
@@ -13,25 +14,56 @@ import qualified System.Process as P
 import Text.Read (readMaybe)
 import qualified Text.Regex.TDFA as R
 import qualified Wasp.SemanticVersion as SV
+import Wasp.Util (indent)
 
+-- | Gets the installed node version, if any is installed, and checks that it
+-- meets Wasp's version requirement.
+--
+-- Returns a string representing the error
+-- condition if node's version could not be found or if the version does not
+-- meet the requirements.
+checkNodeVersion :: IO (Either String SV.Version)
+checkNodeVersion =
```

Comment:
Perhaps a more informative name is `getNodeVersionOrErrorMsg` (or something along those lines).